lib/bandit/headers.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Bandit.Headers do
+  @moduledoc false
+  # Conveniences for dealing with headers
+
+  # credo:disable-for-this-file Credo.Check.Design.AliasUsage
+
+  def get_header(headers, header) do
+    case List.keyfind(headers, header, 0) do
+      {_, value} -> value
+      nil -> nil
+    end
+  end
```

Comment:
You could remove the redundant case on `nil` with `with`. However, without a `@spec` on the function, I do like making the possible returns more obvious.

```suggestion
  def get_header(headers, header) do
    with {_, value} <- List.keyfind(headers, header, 0) do
      value
    end
  end
```

Reply:
I did exactly that originally, and standardized on making the nil case explicit instead. This is a case where the explicitness of a nil case is worth it, I think.